src/classfile/parse.rs

```diff
@@ -38,27 +38,42 @@ impl ClassFileParser {
     pub fn bytes(path: String) -> Result<Vec<u8>> {
         info!("opening classfile '{}' to read bytes", path);
 
-        let mut path = path;
-
-        if path.starts_with("java/") {
+        let path = if path.starts_with("java/") {
             debug!("java stdlib detected, altering path");
+            format!("src/stdlib/{}", path)
+        } else {
+            path
+        };
+        let path_with_ext = format!("{}.class", path);
+
+        debug!("loading bytes from '{}'", path_with_ext);
```

Comment:
This should use the original path so that the logs are factually accurate. Having this use the log with the extension means we will see inaccurate logs for the behaviour of the VM. Additionally, this log should say that we are going to attempt a load from the location, as this is more accurate.